modules/popup/react/lib/usePopupStack.ts

```diff
@@ -40,23 +41,22 @@ import {PopupStack} from '@workday/canvas-kit-popup-stack';
  * useCloseOnEscape(stackRef, closePopup)
  */
 export const usePopupStack = <E extends HTMLElement>(
```

Comment:
I'm not sure I need a generic here at all... This mostly prevents type issues for consumers if they use the wrong ref type...